tests/ops/integration_tests/limiter/test_rate_limiter.py

```diff
@@ -67,9 +72,18 @@ def test_limiter_times_out_when_bucket_full() -> None:
     with pytest.raises(RateLimiterTimeoutException):
         for _ in range(500):
             limiter.limit(
-                key="my_test_key",
-                rate_limit=100,
-                period=RateLimiterPeriod.HOUR,
+                requests=[
```

Comment:
to be a bit more comprehensive, perhaps you could verify that some of your other test cases still work as expected with multiple limits?

i guess i feel like it could also be good to keep the test case here with a single request. i'd prefer to be a bit redundant in the test code and pretty much have single and multiple request versions of most (all?) cases. 
i think these tests are really helpful resources in understanding how the rate limiter functionality, so being comprehensive would not only improve test coverage, but it'd also help clarify the functionality.

Reply:
good point, I will leave it as a single limit. I have to think of a easy way of testing multiple limits with what's currently there.

Reply:
Added a separate test which tests multiple limits now